t/valgrind/workload.d/job

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -e
+
+NJOBS=${NJOBS:-10}
+
+flux submit -n 1 hostname
```

Comment:
it seems you dropped the NJOBS carbon copy when you copied the file over here.  Perhaps testing/playing around and forget to put that back?

Alternate idea.  Instead of adding `mf-priority-plugin-load` in the next commit, just update this test to load the mf-priority-plugin, so now `workload.d/job` is flux-accounting specific.   Also the `--wait` option should be kept so that the job completes before returning.

Reply:
Oh okay! I like that suggestion. Will do that

Reply:
OK, just force-pushed a change to this PR to just have one commit and updates the test to load the priority plugin (I also removed the `job-cancel` test because that didn't have anything flux-accounting specific in it), and added back the `--wait` option.